### PR TITLE
Reword A6g and remove A6g+

### LINKED ISSUE #556: Exception in A6g does not match the main regulation
```
A6g) The judge determines whether the puzzle is solved. They must not make moves or align faces when examining the puzzle. Exception: For Clock, the judge will usually need to pick up the puzzle to verify both faces.
```
Why is the exception about picking up the puzzle when the main regulation only forbids the judge from moving/aligning the puzzle?

### PR DESCRIPTION
**Pull request by pedrosino**
Fixes #556

### CODE REVIEW

**Review by lgarron — APPROVED** [review]
LGTM; I'll wait for another WRC member to review before we merge, though.

I have a slight concern in that we don't specify what happens if a judge *does* touch the puzzle, but I think "must not touch" is still preferable.

File: wca-regulations.md
Comment (Laura-O) [inline wca-regulations.md]: *Before determining* sounds weird to me: when judges must not touch the puzzle before determining, aren't they allowed to touch it while determining?

Comment (lgarron) [reply wca-regulations.md]: "...before they have determined whether..."?

**Comment by theoleinad** [comment]
How about:
+    - A6g) The judge determines whether the puzzle is solved. They must not touch the puzzle until a decision has been taken.

It’s shorter and concise...

**Comment by xsrvmy** [comment]
Maybe:
They must not touch the puzzle until any disputes over misalignment are resolved.

**Comment by lgarron** [comment]
I still prefer "whether they have determined whether to assign a penalty for misalignment".

> They must not touch the puzzle until any disputes over misalignment are resolved.

This sounds okay, but I would prefer this to refer to the judge's decision rather than a dispute by default.

File: wca-regulations.md
Comment (lgarron) [reply wca-regulations.md]:
```suggestion
    - A6g) The judge determines whether the puzzle is solved. They must not touch the puzzle before they have determined whether to assign a penalty for misalignment.
```

**Comment by lgarron** [comment]
@pedrosino: I've added a suggested change to Laura's comment, which I would still prefer to go with. If you're happy with it, could you accept it and we can merge?

**Review by lgarron — APPROVED** [review]

**Pull request merged**
